src/main/java/seedu/address/logic/commands/EditTaskCommand.java

```diff
@@ -83,6 +84,10 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_INVALID_TASK_DATETIME_FOR_RECURRING_TASK);
         }
 
+        if (!taskToEdit.isSameTask(editedTask) && model.hasTask(editedTask)) {
```

Comment:
Could we also throw an error if taskToEdit is the same as edited task- "No changes made"? For this equality use equals() instead of isSameTask() since changing recurrence is still a change.

Reply:
Throws MESSAGE_NO_CHANGE command exception now